src/llama_stack/apis/datatypes.py

```diff
@@ -127,6 +127,7 @@ class Api(Enum, metaclass=DynamicApiMeta):
     files = "files"
     prompts = "prompts"
     conversations = "conversations"
+    file_processor = "file_processor"
```

Comment:
I wonder if this should be plural like `file_processors`? like the APIs above it? This is kind of a nit, but just something to think about!